Description: Enable usage of more mouse buttons
 The number of "extended" mouse buttons by default is 9, which isn't enough in
 this day and age. This patch increases the number of usable mouse buttons.
Author: Klaus Ethgen <Klaus@ethgen.de>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=775570
Forwarded: not-needed
Reviewed-by: Vincent W. Chen <vinmun@gmail.com>
Last-Update: 2015-01-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- libs/defaults.h.orig
+++ libs/defaults.h
@@ -25,7 +25,7 @@
 /* The "extended" buttons do not provide the full functionality because X has
  * no bit mask value for them.  Things like dragging windows don't work with
  * them. */
-#define NUMBER_OF_EXTENDED_MOUSE_BUTTONS   9
+#define NUMBER_OF_EXTENDED_MOUSE_BUTTONS   15
 #if NUMBER_OF_EXTENDED_MOUSE_BUTTONS > 31
 #error No more than 31 mouse buttons can be supported on 32 bit platforms
 #endif
